spongeauth/spongeauth/urls.py

```diff
@@ -34,7 +34,7 @@
     url(r'^admin/', admin.site.urls),
     url(r'^accounts/', include(accounts.urls, 'accounts')),
     url(r'^2fa/', include(twofa.urls, 'twofa')),
-    url(r'^avatar/(?P<username>[A-Za-z_0-9]+)/?$', avatar_for_user, name='avatar-for-user'),
+    url(r'^avatar/(?P<username>[\w-]+)/?$', avatar_for_user, name='avatar-for-user'),
```

Comment:
At this point just give up and use `^/`, I think.

Reply:
I just removed the whole regex check.  `^/` or no check makes no difference.